tests/c/CMakeLists.txt

```diff
@@ -10,15 +10,13 @@ endif()
 if(BUILD_TESTS)
     add_executable(unittests
         channel_tests.cpp
-        cscore_tests.cpp
-        csound_circular_buffer_test.cpp
```

Comment:
I see cscore was dropped, but it seems like csound_circular_buffer_test.cpp should be restored here.

Reply:
added back.